index.html

```diff
@@ -156,6 +156,22 @@
           </select>
     </div>
 
+    <div class="input-group mt-2">        
+        <i class="input-group-text bi bi-123"></i>
```

Comment:
Dear Jack,

The new field added has the field name displayed as "123". Please change the code accordingly to display the field name as something meaningful such as the Number of Sensors to be Detected or anything appropriate.

Also, the placeholder text is too long(an empty field would be fine). I think this text could be added as an info tab.

Thanks
SaiSri.